beater/client.go

```diff
@@ -29,7 +29,11 @@ func isServerUp(secure bool, host string, numRetries int, retryInterval time.Dur
 
 	for i := 0; i <= numRetries; i++ {
 		if check() {
-			logp.Info("HTTP Server ready")
+			secureInfo := ""
+			if secure {
+				secureInfo = " (secure mode activated)"
+			}
```

Comment:
building `secureInfo` can be done outside the for loop.  Also good with removing it altogether.

Reply:
Agree with building it outside the for loop. Not agree with removing it. IMO, logged that the server is in secure mode is relatively important to know ?